src/design-tokens/tier-1-definitions/outline.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
box.json?

Reply:
Like comment above, we can consider consolidating this with `borders` but would be a separate issue